build.sbt

```diff
@@ -18,6 +18,7 @@ libraryDependencies ++= Seq(
   "org.apache.commons" % "commons-compress" % "1.18",
   // for jdkpackager
   "org.apache.ant" % "ant" % "1.10.5",
+  "com.github.eldis" % "tool-launcher" % "0.2.1",
```

Comment:
Can you add a comment why we need this dependency? :)

Also. How did you find this lib? It looks rather new and built for a specific use case. Will this work in the future?

Reply:
It is new, and it is, indeed built for this exact use case. :smile: I've put it into the company github/sonatype organization so that I don't have to go through the trouble of establishing an organization for myself.

The library is pure Java, and has zero dependencies, so short of Java breaking the `ToolProvider` API, I don't see why this wouldn't work for the foreseeable future. And, since the library itself is a hundred lines of Java code under the MIT license, it should be reasonably easy to update/fix/rewrite it if the unthinkable happens. That is, if we stil have to support Java 8 by that point. :smile: